transport/http/pom.xml

```diff
@@ -163,6 +163,11 @@
             <groupId>org.kaazing</groupId>
```

Comment:
Why do you need to change this file? This PR should only affect the amqp service.